packages/block-library/src/post-comment-edit/edit.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * WordPress dependencies
+ */
+import { InspectorControls, useBlockProps } from '@wordpress/block-editor';
+import { PanelBody, ToggleControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+export default function Edit( { attributes, setAttributes } ) {
+	const { openInNewTab } = attributes;
+
+	return (
+		<>
+			<InspectorControls>
+				<PanelBody title={ __( 'Link settings' ) }>
+					<ToggleControl
+						label={ __( 'Open in a new tab' ) }
+						checked={ openInNewTab }
+						onChange={ () =>
+							setAttributes( {
+								openInNewTab: ! openInNewTab,
+							} )
+						}
+					/>
+				</PanelBody>
+			</InspectorControls>
+			<div { ...useBlockProps() }>
+				<a href="#edit-comment-pseudo-link">{ __( 'Edit' ) }</a>
```

Comment:
Clicking the link in the editor seems to cause an unusual crash and some console errors, not sure what's happening, but it's probably unrelated to this PR. Thought I'd flag it though.

Reply:
Nice catch! It could cause some pretty nasty behaviour. I've created an issue for that: https://github.com/WordPress/gutenberg/issues/30967

I wouldn't know how to adres this in the scope of this block though besides removing the href. But removing the href has its own issues.